draft-ietf-wish-whip.md

```diff
@@ -638,9 +638,9 @@ The Designated Expert (DE) is expected to ascertain the existence of suitable do
 
 The DE is also expected to check the clarity of purpose and use of the requested registration.
 
-Additionally, the DE must verify that any request for one of these registrations has been made available for review and comment within the IETF. the DE will post the request to the WebRTC Ingest Signaling over HTTPS (wish) Working Group mailing list (or a successor mailing list designated by the IESG). 
+Additionally, the DE must verify that any request for one of these registrations has been made available for review and comment by posting the request to the WebRTC Ingest Signaling over HTTPS (wish) Working Group mailing list (or a successor mailing list designated by the IESG). 
```

Comment:
```suggestion
Additionally, the DE must verify that any request for one of these registrations has been made available for review and comment by posting the request to the WebRTC Ingest Signaling over HTTPS (wish) Working Group mailing list. 
```

Reply:
The chairs are not going to ask for the wish mailing list to close. For this protocol it's way too much to close the WG mailing list and start another one just for registrations.